src/tests/run-study-tests/check_on_results/output_compare.py

```diff
@@ -85,9 +85,13 @@ def compare_simulation_files(simulation_files, tol):
     at_least_one_diff = False
     col_name_where_diff = ""
     for file_pair in simulation_files:
+        rows_to_skip = [0,1,2,3,5,6]
```

Comment:
We used to have regressions on **details** output files : indeed, adding a new output variables for every cluster results in adding a new column in details files. This new column's title is the name of the cluster, again (as for every variable associated to a cluster).
So now, for a details file, we give the columns the subtitles (Units) as their names.